mica/archive/aca_l0.py

```diff
@@ -42,6 +42,25 @@
             'arc5gl_query': 'ACA0',
             'fileglob': '*fits.gz'}
 
+ACA_DTYPE_2 = (('TIME', '>f8'), ('QUALITY', '>i4'), ('MJF', '>i4'),
```

Comment:
These probably both need comments now.  And should we capture the new fields if available (PIXTLM BGDTYP)?